HousingRepairsOnlineApi/UseCases/SaveRepairRequestUseCase.cs

```diff
@@ -33,6 +33,7 @@ public async Task<Repair> Execute(string repairType, RepairRequest repairRequest
                 repairRequest.Issue?.Value);
             var repair = new Repair
             {
+                RepairType = repairType,
```

Comment:
This comment follows [this](https://github.com/Newarkandsherwood/housing-repairs-online-api/pull/30/files#diff-b15b8145c57692b872e41e985c150d39ae0ce9eb6f2126db29e79cebb5cd68b5R1) one on `RepairTypeHelper`.

I think(!) this usage uses the (pre-existing) `RepairType`, which means that when saving we are using one of the repair type helpers (pre-existing) and when querying we are using another (introduced in this PR). 
They should be consolidated.